apps/fabric-website/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { Link, DefaultButton, mergeStyles } from 'office-ui-fabric-react';
+import {
+  trackEvent,
+  EventNames,
+  randomEntry,
+  getSiteArea,
+  Page,
+  IPageProps,
+  IPageSectionProps
+} from '@uifabric/example-app-base/lib/index2';
+import { SiteDefinition } from '../../SiteDefinition/index';
+import { topNavHeight } from '../../styles/constants';
+
+const illustrations = [
+  'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/images/error/error1.svg',
+  'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/images/error/error2.svg',
+  'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/images/error/error3.svg'
+];
+
+const rootClass = mergeStyles({
+  backgroundRepeat: 'no-repeat',
+  backgroundSize: '480px',
```

Comment:
What will happen with this on small screens?

Reply:
I've added responsive styles.